modifiers/caliper/modifier.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from ramble.modkit import *
```

Comment:
import * is triggering the CI failure.  Please see under actions/style/lint.  The solution is to import the specific modules you need instead of *.

Reply:
Relaxed this requirement, consistent with ramble